core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala

```diff
@@ -424,6 +445,27 @@ class EventsByTagSpec extends AbstractEventsByTagSpec(EventsByTagSpec.config) {
       })
     }
 
+    "include timestamp in EventEnvelope" in {
```

Comment:
that test name is already in use

Reply:
more than that, we have it twice

Reply:
it's two different test sections, one for `currentEventsByTag` and another for `eventsByTag`

Reply:
Ah, sorry. I haven't expanded the full context in GitHub.